applications/nrf5340_audio/src/bluetooth/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bt_le_audio_tx.h"
+
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/zbus/zbus.h>
+#include <../subsys/bluetooth/audio/bap_iso.h>
+
+#include "nrf5340_audio_common.h"
+
+/* TODO: Remove when a get_info function is implemented in host */
+#include <../subsys/bluetooth/audio/bap_endpoint.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(le_audio, CONFIG_BLE_LOG_LEVEL);
+
+ZBUS_CHAN_DEFINE(sdu_ref_chan, struct sdu_ref_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
+		 ZBUS_MSG_INIT(0));
+
+#ifdef CONFIG_BT_BAP_UNICAST_SERVER
+#define SRC_STREAM_COUNT CONFIG_BT_ASCS_ASE_SRC_COUNT
+#elif CONFIG_BT_BAP_UNICAST_CLIENT
+#define SRC_STREAM_COUNT CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT
+#elif CONFIG_BT_BAP_BROADCAST_SOURCE
+#define SRC_STREAM_COUNT CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT
+#else
+#define SRC_STREAM_COUNT 0
+#endif
```

Comment:
Have to think of something smart once/if we create an app that can do both BIS and CIS